### PR TITLE
Feature/liked user view

### PR DESCRIPTION
**Pull request by magovansh9**
Created a view that shows the users who have liked a post.

### CODE REVIEW

File: app/src/src/components/views/LikedUserView.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+// import ChatBubbleIcon from "@material-ui/icons/ChatBubble";
```
Comment (deandrebaker) [inline app/src/src/components/views/LikedUserView.jsx:3]: If this is not used, we can delete this import

File: app/src/src/components/views/LikedUserView.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import {
+  configureFonts,
+  DefaultTheme,
+  Provider as PaperProvider,
+} from 'react-native-paper';
+
+const LikedUserView = (props) => {
+  const { first_name, profile_photo } = props;
```
Comment (deandrebaker) [inline app/src/src/components/views/LikedUserView.jsx:11]: We can get rid of line 11 by modify line 10 to be
`const LikedUserView = ({ first_name, profile_photo }) => {`
This helps with intellisense

File: app/src/src/screens/likes/LikesScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { StyleSheet, View } from 'react-native';
+import Header from '../../components/views/Header';
+import FeedContainer from '../../components/containers/FeedContainer';
+import CreateItemModal from '../../components/modals/CreateItemModal';
+import { CommentsEndpoint } from '../../utils/endpoints';
+import CommentView from '../../components/views/CommentView';
+import UsersList from '../../components/common/UsersList';
+
+const endpoints = [CommentsEndpoint];
+const itemViews = [(item) => <CommentView {...item} />];
```
Comment (deandrebaker) [inline app/src/src/screens/likes/LikesScreen.jsx:11]: Why do these say comments?

File: app/src/src/components/views/CommentBar.jsx [hunk 4]
```diff
@@ -60,7 +60,10 @@ const CommentBar = ({ item, contentType, endpoint }) => {
         <View style={styles.likesAndCommentsText}>
           <Text
             onPress={() => {
-              // TODO : Show who has liked this post onPress
+              nav.navigation.navigate(routes.comments, {
```
Comment (deandrebaker) [inline app/src/src/components/views/CommentBar.jsx:63]: Why does this say comments?

**Review by deandrebaker — CHANGES_REQUESTED** [review]
@magovansh9 looking good so far! I see a lot of references to comments, but this ticket was for likes. Was that intentional?

File: app/src/src/components/views/LikedUserView.jsx (hunk 1 above)
Comment (magovansh9) [reply app/src/src/components/views/LikedUserView.jsx:3]: 👍

File: app/src/src/components/views/LikedUserView.jsx (hunk 2 above)
Comment (magovansh9) [reply app/src/src/components/views/LikedUserView.jsx:11]: 👍

File: app/src/src/screens/likes/LikesScreen.jsx (hunk 3 above)
Comment (magovansh9) [reply app/src/src/screens/likes/LikesScreen.jsx:11]: Most of the functionality is taken from the way the comments screen is rendered. As we do not have a likes endpoint at the moment, the PR was for you and Trent to make initial suggestions on how we can implement the screen, referring back to how comments are rendered.

**Pull request closed without merging**